### ISSUE TITLE
Suppress maven-dependency-plugin

### ISSUE THREAD

**Issue by tesshucom** [tesshucom/jpsonic#1100]
There were so many issues right after 16 was released, but many of them seem to have been resolved. It may be time to consider Bump.

<details>
<summary>Some nonconformities exist and need to be fixed.</summary>

#### Delete jukebox

#1107

#### QueryFactoryTest

Is it due to ToStringBuilder#reflection ... ? We need to look into the details of the error a bit. It may be possible to avoid this by modernizing the assert in the list.

```
[ERROR] testGetRandomAlbumsId3  Time elapsed: 0.226 s  <<< ERROR!
java.lang.reflect.InaccessibleObjectException: Unable to make field private static final long java.util.Arrays$ArrayList.serialVersionUID accessible: module java.base does not "opens java.util" to unnamed module @6b71769e
        at com.tesshu.jpsonic.service.search.QueryFactoryTest.testGetRandomAlbumsId3(QueryFactoryTest.java:157)

[ERROR] testGetMediasByGenre  Time elapsed: 0.465 s  <<< ERROR!
java.lang.reflect.InaccessibleObjectException: Unable to make field private static final long java.util.Arrays$ArrayList.serialVersionUID accessible: module java.base does not "opens java.util" to unnamed module @6b71769e
        at com.tesshu.jpsonic.service.search.QueryFactoryTest.testGetMediasByGenre(QueryFactoryTest.java:167)

[ERROR] testGetAlbumId3sByGenre  Time elapsed: 0.007 s  <<< ERROR!
java.lang.reflect.InaccessibleObjectException: Unable to make field private static final long java.util.Arrays$ArrayList.serialVersionUID accessible: module java.base does not "opens java.util" to unnamed module @6b71769e
        at com.tesshu.jpsonic.service.search.QueryFactoryTest.testGetAlbumId3sByGenre(QueryFactoryTest.java:183)

[ERROR] testGetRandomAlbums  Time elapsed: 0.005 s  <<< ERROR!
java.lang.reflect.InaccessibleObjectException: Unable to make field private static final long java.util.Arrays$ArrayList.serialVersionUID accessible: module java.base does not "opens java.util" to unnamed module @6b71769e
        at com.tesshu.jpsonic.service.search.QueryFactoryTest.testGetRandomAlbums(QueryFactoryTest.java:148)
```

</details>

_____

The fix inside the project is probably successful, but there seems to be a problem with the maven-dependency-plugin. Requires update to 3.2.0 for JDK16. On the other hand, 3.2.0 has a regression. 

 - [Non-test dependency reported as Non-test scoped test only dependency](https://issues.apache.org/jira/browse/MDEP-753)
 - ['Dependency not found' with 3.2.0 and Java-17 while analyzing](https://issues.apache.org/jira/browse/MDEP-759)

Forced compilation is possible locally, and it is possible to check the operation of JDK16 and 17-ea. However, since the workflow does not pass normally, merging to master is currently impossible. This is FATAL. There is no workaround.

_____

Temporarily suppress maven-dependency-plugin in the ​part of the workflow.

 - There are only a few days left until the release of the next LTS, java17. 
 - The maven-dependency-plugin bug is unfortunate, but it's not a good idea to block all other validations by the bug. Therefore, the execution of maven-dependency-plugin is suppressed in the verification of a specific version..
 - Workflow verify has different configurations for lts (11) and experiment (12, 13, 14, 15, 16, 17-ea). lts validates on Windows and Linux and checks for dependencies. The malfunction was confirmed in jkd16, 17-ea,  windows. Therefore, dependency check is performed in LTS Linux, and it is suppressed in other cases.
 - Only dependency checking is suppressed. Testing is required, and run on all versions.
 - Power shell has a special specification for argument -D. Command is different from Linux.

Maven-dependency-plugin bugs will be tracked and workflow configuration will be restored as soon as they are fixed. This issues will remain open until they are resolved.

https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-dependency-plugin

**Comment by tesshucom** [comment]
AudioPlayer creates and validates a local library derived from the currently used 0.2.5. If there is no problem, it will be merged, but if the operation is confirmed, check with the author if there is a plan to update.

**Comment by tesshucom** [comment]
All the bugs that were blockers have been fixed, and 3.3.0 has been released. Check the fix and repair the workflow if there is no problem.
